extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationSearch.js

```diff
@@ -7,16 +7,14 @@ $(function ($) {
 		placeholderText = $searchInput.attr('placeholder'),
 		activeSearchClass = 'wds-search-is-active';
 
-	if ($searchInput.val().length === 0) {
```

Comment:
I moved this down to add structure to the file:
* vars,
* function defs,
* listeners,
* code that runs upon script load.